contracts/SportMarkets/SportPositions/SportPositionalMarketManager.sol

```diff
@@ -141,6 +128,17 @@ contract SportPositionalMarketManager is Initializable, ProxyOwned, ProxyPausabl
         ISportPositionalMarket(_market).setPaused(_paused);
     }
 
+    function updateDatesForMarket(address _market, uint256 _newStartTime) external override {
+        require(
+            msg.sender == owner || msg.sender == theRundownConsumer || whitelistedAddresses[msg.sender],
+            "Invalid caller"
+        );
+
+        uint expiry = _newStartTime.add(expiryDuration);
+
+        ISportPositionalMarket(_market).updateDates(_newStartTime, expiry);
+    }
```

Comment:
better to trigger in the Manager `DatesUpdatedForMarket(_market, _maturity, _expiry)` mainly for subgraph

Reply:
done